pom.xml

```diff
@@ -67,6 +67,12 @@
         <artifactId>ibis-adapterframework-ifsa</artifactId>
         <version>7.6-SNAPSHOT</version>
       </dependency>
+      <dependency>
+        <groupId>com.ing</groupId>
+        <artifactId>ifsa-jms-xa</artifactId>
+        <version>22.10.055</version>
+        <scope>provided</scope>
+      </dependency>
```

Comment:
Gelieve geen proprietary dependencies in de parent pom. Deze dependency is zeer specifiek voor de IFSA module, dat hoort hij dan ook thuis.